cli/cli-flags.js

```diff
@@ -205,12 +205,16 @@ function getYargsParser(manualArgv) {
         type: 'string',
         describe: 'The path to the budget.json file for LightWallet.',
       },
+      'disable-full-page-screenshot': {
```

Comment:
This is a bit long, I think we can shorten to `--disable-screenshot`

Reply:
There are so many screenshots that could refer to.